cfgov/jinja2/v1/_includes/molecules/notification.html

```diff
@@ -38,7 +38,7 @@
   <div class="m-notification
               {{ type }}
               {{ 'm-notification__visible' if is_visible else '' }}">
-      {% include 'icons/' ~ icon ~ '.svg' if icon else '' %}
+      {{ svg_icon(icon) }}
```

Comment:
@chosak Do we need to wrap this in a condition or will the template tag gracefully handle it?